### ISSUE TITLE
Pre-mounted configurations LVM on LUKS failed: Could not detect root at mountpoint

### ISSUE THREAD

**Issue by Goosegit11** [archlinux/archinstall#2925]
### Which ISO version are you using?

2024-11-01

### The installation log

```json
https://0x0.st/X52K.log
```

### describe the problem

conputer

**Comment by svartkanin** [comment]
I'm not sure if this setup is supported yet, I need to see if pre-mounted LVMs would be recognized correctly

**Comment by Goosegit11** [comment]
Yeah seems like there's no support for that, sadly. And with LVM (beta) option I get error too so no way at all to install Arch on LVM with archinstall

**Comment by svartkanin** [comment]
What is the error from the LVM installation (could you file a different issue with that)

**Comment by Goosegit11** [comment]
it's #2924
